spds/constraints.md

```diff
@@ -4,4 +4,71 @@ title: Constraints
+
+![example-constraint.png](..%2Fimages%2Fconstraints%2Fexample-constraint.png)
+
+The picture below depicts the behavior if a cooldown constraint is defined for a time period of 4 time units and with maximum scaling operation within that period of 0.
+Once the policy is enacted once and an adjustment is made there is no further adjustment for the next 4 time units.
+![example-cooldown-1.png](..%2Fimages%2Fconstraints%2Fexample-cooldown-1.png)
+
+In case the max number of scaling operations is set to 1, after the first adjustment, there is one additional operation allowed within the cooldown period but no more than one.
+![example-cooldown-2.png](..%2Fimages%2Fconstraints%2Fexample-cooldown-2.png)
+
+Cooldown constraints are applicable to the policy and help to allow the system to stabilize after another adjustment is made so that the effects are better observable.
+
+### Interval Constraint
+An interval constraint is defined by a time offset and a time period in which the policy is switched to an inactive state.
+For the same example, the picture below defines a time offset of 2 time units and a time period of 5 time units.
+During the offset period, the policy is active and adjustments are made normally. 
+After offset ends, the inactivity period starts that lasts for the specified _intervalDuration_.
+
+After the inactivity period ends and the simulation continues, the policy is switched to an active state again and adjustments are made normally.
+However, if the _repeat_ flag is set to true, the interval constraint is applied again with the _offset_ and _intervalDuration_ (shown with a dashed line in the figure).
+
+![example-interval.png](..%2Fimages%2Fconstraints%2Fexample-interval.png)
+
+### Thrashing Constraint
+
+Another temporal constraints that is applicable to the whole [TargetGroup](../docu/#targetsTargetGroup) and restricts the behavior of all the policies applying to it. 
+The main objective behind the thrashing constraint is to avoid oscillations in the system. 
+The thrashing constraint is defined by a minimum time period between two consecutive adjustments with opposite directions.
+The motivation behind defining a thrashing constraint can be the stability of the system and the billing type of the cloud provider.
+Often when resources are provisioned it does not make any sense to deprovision them after a short period of time while you are billed for the next 10 minutes. 
+Hence, the modeller can specify a Thrashing constraint.
+
+The picture below shows the behavior of a policy without a thrashing constraint in which a consecutive scale in is occurring after a scale out.
+![constraint-thrashing.png](..%2Fimages%2Fconstraints%2Fconstraint-thrashing.png)
+
+When adding a Thrashing constraint, the modeler can avoid such an oscillation. The picture below shows the behavior of the same policy with a thrashing constraint of 5 time units.
+![constraint-thrashing-2.png](..%2Fimages%2Fconstraints%2Fconstraint-thrashing-2.png)
```

Comment:
it bugs me, that these pictures are inconsistent with those from the previous sections. 
inconsisten as in : in the previous section, the constraint where depicted on the time line, and the braces for constraint related information was above the picutre, but now the constraints stuff is below the picture and the contraint is not on the timeline. 
EDIT : content wise the picture is fine!! i'm just nagging about the presentation.